static/js/redux/actions/modal_actions.jsx

```diff
@@ -156,11 +156,20 @@ export const toggleSeparateAccountsModal = () => ({
   type: ActionTypes.TOGGLE_SEPARATE_ACCOUNTS_MODAL,
 });
 
-// MOCK MODAL
+// SIS MODAL
 export const triggerSISImportDataModal = () => ({
   type: ActionTypes.TRIGGER_SIS_IMPORT_DATA_MODAL,
 });
 
 export const toggleSISImportDataModal = () => ({
   type: ActionTypes.TOGGLE_SIS_IMPORT_DATA_MODAL,
 });
+
+// ADD ADVISOR MODAL
+export const triggerAddAdvisorModal = () => ({
+  type: ActionTypes.TRIGGER_ADD_ADVISOR_MODAL,
+});
+
+export const toggleAddAdvisorModal = () => ({
```

Comment:
Duplicate, needs to be removed. Look at the full file to see the other value.

Reply:
How is this a duplicate ?

Reply:
Oh lol wait